stable/enterprise/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: enterprise
-version: "3.5.2"
+version: "3.5.3"
```

Comment:
Since this is a new feature, though backwards compatible, I think this is "3.6.0" not a patch update.

Reply:
@zhill since v5.16.0 took the chart to 3.6.0, should I then bump this to 3.7.0?